avocado/utils/lv_utils.py

```diff
@@ -38,20 +38,21 @@ class LVException(Exception):
 
 def get_diskspace(disk):
     """
-    Get the entire disk space of a given disk.
+    Get the entire disk space of a given disk(s).
 
-    :param str disk: name of the disk to find the free space of
+    :param str disk: name of the disk(s) to find the free space of
```

Comment:
`name of the disk(s), comma separated, to find the free space of`

Reply:
@narasimhan-v Abdul said the inputs will be of space separated, can we discus in group and conclude on this?

Reply:
@Naresh-ibm Right, space separated. My mistake.